src/tokens/tokens.interfaces.ts

```diff
@@ -57,18 +57,36 @@ export enum TokenType {
   NONFUNGIBLE = 'nonfungible',
 }
 
+const trackingIdDescription =
+  'Optional ID provided by the client for correlating related events. There are no ' +
+  'restrictions on this field - it will not be inspected by the server, but will be ' +
```

Comment:
Wonder if there should be a length restriction?

Reply:
I'll change the verbage to be a little more general, but probably will wait and take a pass on stricter length/other validations on all fields a little further down the road.